man/systemd.network.xml

```diff
@@ -1004,6 +1004,26 @@ DuplicateAddressDetection=none</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MulticastIGMPVersion=</varname></term>
+        <listitem>
+          <para>Configures IPv4 Multicast IGMP Version to be used, and controls the value of
+          <filename>/proc/sys/net/ipv4/conf/<replaceable>INTERFACE</replaceable>/force_igmp_version</filename>.
+          Defaults to unset, and the sysctl value will be unchanged. Takes one of <literal>no</literal>,
+          <literal>v1</literal>, <literal>v2</literal>, or <literal>v3</literal>.
+          When <literal>no</literal>, no enforcement of an IGMP version, IGMPv1/v2 fallback allowed, will back to IGMPv3 mode again
```

Comment:
`no enforcement of an IGMP version will be applied, IGMPv1/v2 fallback are allowed,`